backend/de.metas.payment.esr/src/main/java/de/metas/payment/esr/api/impl/ESRImportDAO.java

```diff
@@ -105,12 +84,25 @@ public void saveOutOfTrx(@NonNull final I_ESR_Import esrImport)
 		InterfaceWrapperHelper.save(esrImport, ITrx.TRXNAME_None);
 	}
 
+	@Override
+	public void saveOutOfTrx(@NonNull final I_ESR_ImportFile esrImportFile)
+	{
+		InterfaceWrapperHelper.save(esrImportFile, ITrx.TRXNAME_None);
```

Comment:
use saveRecord.

also, why are u using ITrx.TRXNAME_None ?

Reply:
The esrImport itself used to be saved like this, out of trx, here: 

![image](https://user-images.githubusercontent.com/16001442/127041753-8d250218-2862-4892-95f1-f1c5182812e9.png)


So I kept the behaviourfor the importFile as well, since the file is another step between the import and its lines, but plays a role that  is equivalent with  the import for the lines of a certain file from a zip.

Is there another way of saving out of trx, without using TRX_None?

Reply:
ahh ok, then leave it like it is